doc/mainpage.dox

```diff
@@ -55,20 +55,20 @@ that support time, spectra and tables have yet been wrapped. The python wrapping
     @ref Frame "Frames" are mutable, so all frame-specific attributes have setters as well as getters.
 - @ref Channel "Channels" are constructed with a @ref Stream; subclasses are available for files and strings,
     and it is easy (in C++, but not Python) to construct a @ref Stream to use standard in and/or out.
-- astshim manages memory using standard smart pointers. Thus the following
-    AST functions are not wrapped: `astAnnul`, `astBegin`, `astClone`, `astDelete`,
-    `astEnd`, and `astExport`.
+- astshim manages memory using C++ smart pointers. Thus the following AST functions are not wrapped:
+    `astAnnul`, `astBegin`, `astClone`, `astDelete`, `astEnd`, and `astExport`.
 - Methods that output floating point data have `AST__BAD` replaced with `nan`.
 
 ### Smaller differences (not a complete list):
 
-- Methods such as @ref Object.set do not support printf formatting and extra arguments.
+- Methods such as @ref Object.set and the attributes argument of constructors do not support
```

Comment:
Good to know. But cleaning up all the usage in this package would be quite a chore. I'm inclined to leave it as explicit.